method_comparison/text_generation_benchmark/README.md

```diff
@@ -0,0 +1,179 @@
+## Base Model Inference Caching
+
+The benchmarking suite uses a separate script, `run_base.py`, to measure base model inference times and save results for reuse. This should be run once per model configuration to avoid redundant computations and ensure consistent baseline metrics for all PEFT experiments.
+
+**Usage:**
+```bash
+python run_base.py experiments/lora/lora_r8
```

Comment:
Calling the script like this is not possible, as the argument parser does not recognize the argument. Also, the base model results should not depend on any specific experiment. Instead, it should read the `default_benchmark_params.json`.